docs/source/documentation.rst

```diff
@@ -105,34 +62,9 @@ Virtual environments are a tool to separate the Python environment and packages
             We don't document them publically because they are rare and the distraction will sew confusion.
             People in these situations can ask us for help.
 
+For Windows systems, run::
 
-To setup the Python virtual environment with all the required packages, run the following command::
-
-    conda env create
-
-.. WARNING :: For some users, the installation may take up to 30 minutes as many dependencies have shared subdependencies, and resolving these potential conflicts takes time. If that's the case, we encourage you to take a break from your screen and go for a walk while listening to the `AxonDeepSeg Spotify playlist <https://open.spotify.com/playlist/27LVNnfhTKjVOli6bPCaV5?si=OydcwxoOSamwCsg3xcqybw>`_.
-
-Then, activate your virtual environment::
-
-    conda activate ads_venv
-
-.. NOTE :: To switch back to your default environment, run:
-  ::
-
-       conda deactivate
-
-Once your virtual environment is installed and activated, install the AxonDeepSeg software with the following commands::
-
-    pip install -e . plugins/
-
-.. NOTE :: To update an already cloned AxonDeepSeg package, pull the latest version of the project from GitHub and re-install the application:
-   ::
-
-        cd axondeepseg
-        git pull
-        pip install -e . plugins/
-
-.. WARNING :: When re-installing the application, the model folders in ``AxonDeepSeg/models`` will be deleted and re-downloaded. Please do not store valuable data in these folders.
+    install_ads.bat
 
 
 Testing the installation
```

Comment:
I can't highlight this line (it's outside the diff) but on line 72, there is a warning that the virtual environment needs to be activated to run the quick test.

This is no longer necessary, since `axondeepseg_test` is now on the PATH after installation.

Reply:
Deleted!